omnibus/files/private-chef-cookbooks/private-chef/resources/pg_sqitch.rb

```diff
@@ -15,7 +15,6 @@
 #
 default_action :deploy
 
-attribute :name,           kind_of: String, required: true, name_attribute: true
```

Comment:
it's possible we were abusing the name property here, but if that's the case we need to stop calling this resource with a name property specified.